### PR TITLE
sci-physics/hepmc: add 3.2.5

### PR DESCRIPTION
**Pull request by APN-Pucky**
Signed-off-by: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>

### CODE REVIEW

**Comment by APN-Pucky** [comment]
Since this introduced the slot `sci-physics/hepmc-3.1.5:3`. The reverse-dependent packages `sci-physics/pythia` and `sci-physics/thepeg`. Might also need to be changed to depend on `sci-physics/hepmc-2.06.09-r2:0` (Should this be put in `SLOT=2`?).

**Comment by amadio** [comment]
Thank you for adding HepMC3, I've been meaning to add it for a while, but the CMake build system was a bit finicky and I had little time for it. I'm away from my machine for the next week and a half, but if it's not merged by the time I'm back, I will give this a go and merge.

File: sci-physics/hepmc/metadata.xml [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+<maintainer type="person" proxied="yes">
+  <email>alexander@neuwirth-informatik.de</email>
+  <name>Alexander Puck Neuwirth</name>
+</maintainer>
+<maintainer type="project" proxied="proxy">
```
Comment (AndrewAmmerlaan) [inline sci-physics/hepmc/metadata.xml:8]: CC @gentoo/proxy-maint for new proxy-maintainer.

File: sci-physics/hepmc/hepmc-3.2.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+IUSE="doc test examples python root"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="python? ( ${PYTHON_DEPS} )"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	root? ( sci-physics/root:= )
+	doc? (
+		app-doc/doxygen
```
Comment (AndrewAmmerlaan) [inline sci-physics/hepmc/hepmc-3.2.5.ebuild:29]:
```suggestion
		app-doc/doxygen[dot]
```
*      sh: line 1: dot: command not found

**Comment by AndrewAmmerlaan** [comment]
Very nice work, Thanks :+1: 

> Since this introduced the slot `sci-physics/hepmc-3.1.5:3`. The reverse-dependent packages `sci-physics/pythia` and `sci-physics/thepeg` might also need to be changed to depend on previous `SLOT=0` `sci-physics/hepmc-2.06.09-r2:0` (Should this be put in `SLOT=2`?).

I have taken care of this, pythia and thepeg have been bumped. Old versions depend on hepmc:2, the new versions depend on hepmc:3.

**Comment by amadio** [comment]
Thanks, @AndrewAmmerlaan!

**Pull request closed without merging**